src/pybind/feat/wave_reader_pybind.cc

```diff
@@ -0,0 +1,83 @@
+// pybind/feat/wave_reader_pybind.cc
+
+// Copyright 2019   Microsoft Corporation (author: Xingyu Na)
+
+// See ../../COPYING for clarification regarding multiple authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY IMPLIED
+// WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABLITY OR NON-INFRINGEMENT.
+// See the Apache 2 License for the specific language governing permissions and
+// limitations under the License.
+
+#include "feat/wave_reader_pybind.h"
+#include "feat/wave-reader.h"
+#include "util/table-types.h"
+
+using namespace kaldi;
+
+void pybind_wave_reader(py::module& m) {
+  m.attr("kWaveSampleMax") = py::cast(kWaveSampleMax);
+
+  py::class_<WaveInfo>(m, "WaveInfo")
+      .def("IsStreamed", &WaveInfo::IsStreamed)
+      .def("SampFreq", &WaveInfo::SampFreq)
+      .def("SampleCount", &WaveInfo::SampleCount)
+      .def("Duration", &WaveInfo::Duration)
+      .def("NumChannels", &WaveInfo::NumChannels)
+      .def("BlockAlign", &WaveInfo::BlockAlign)
+      .def("DataBytes", &WaveInfo::DataBytes);
+
+  py::class_<WaveData>(m, "WaveData")
+      .def("Duration", &WaveData::Duration)
+      .def("Data", &WaveData::Data);
+
+  py::class_<SequentialTableReader<WaveHolder>>(m, "SequentialWaveReader")
+      .def(py::init<>())
+      .def(py::init<const std::string&>())
```

Comment:
It's better to add the argument here so that we can pass key arguments instead of positional argument in python.

For example

```cpp
.def(py::init<const std::string&>(), py::arg("rspecifier"))
```